src/D2L.CodeStyle.Analyzers/Async/Generator/AsyncToSyncMethodTransformer.cs

```diff
@@ -11,6 +11,9 @@ internal sealed class AsyncToSyncMethodTransformer : SyntaxTransformer {
 		CancellationToken token
 	) : base( model, token ) { }
 
+	// Need to disable D2L0018 in the method if we add Task.Run() to syncify something
+	private bool disableTaskRunWarningFlag;

```

Comment:
```suggestion
	private bool m_disableTaskRunWarningFlag;
```

(you'll need to update it elsewhere though)